server/tournament/models.py

```diff
@@ -0,0 +1,307 @@
+from typing import Any
+
+from django.db import models
+from django.db.models.signals import m2m_changed
+from django.dispatch import receiver
+from django.utils.translation import gettext_lazy as _
+from django_prometheus.models import ExportModelOperationsMixin
+
+from server.core.models import Player, Team, UCPerson
+from server.series.models import Series, SeriesRegistration
+from server.utils import default_invitation_expiry_date, slugify_max
+
+
+class Event(ExportModelOperationsMixin("event"), models.Model):  # type: ignore[misc]
+    class Type(models.TextChoices):
+        MIXED = "MXD", _("Mixed")
+        OPENS = "OPN", _("Opens")
+        WOMENS = "WMN", _("Womens")
+
+    title = models.CharField(max_length=255)
+    start_date = models.DateField()
+    end_date = models.DateField()
+    team_registration_start_date = models.DateField()
+    team_registration_end_date = models.DateField()
+    player_registration_start_date = models.DateField()
+    player_registration_end_date = models.DateField()
+    ultimate_central_id = models.PositiveIntegerField(unique=True, null=True, blank=True)
+    ultimate_central_slug = models.SlugField(max_length=200, default="unknown")
+    location = models.CharField(max_length=255, default="unknown")
+    type = models.CharField(max_length=3, choices=Type.choices, default=Type.MIXED)
+    slug = models.SlugField(null=True, blank=True, db_index=True)
+    max_num_teams = models.PositiveSmallIntegerField(blank=True, null=True)
+    series = models.OneToOneField(Series, on_delete=models.SET_NULL, blank=True, null=True)
+    qualifier_event_id = models.OneToOneField(
```

Comment:
this is not needed for now I think.. And this ideally should be list as South Regionals will have TN, KA, KL Sectionals as qualifier event..

So yeah I dont think itll be much useful now anyway?